### DISCUSSION TITLE
Groq API Support

### DISCUSSION THREAD

**Discussion by NielSync** [Taewan-P/gpt_mobile#68] [Ideas]
Hi! Please consider adding support for the Groq API, please. Otherwise thanks for the app :)

**Comment by zuwf** [comment]
It's really helpful if we get groq support..
Because it's the fastest server to deliver faster response

**Comment by Taewan-P** [comment]
@NielSync @zuwf I just added Groq support via #103. It will be in the next release.

I am really surprised of the API speed... Really impressive.

Thanks for your idea/feedbacks & using the app.

**Reply by NielSync** [reply]
Could you also add DeepSeek API support? I tried to add it via OpenAI settings but it just doesn't work. Thanks!

**Reply by Taewan-P** [reply]
<img width="1095" alt="image" src="https://github.com/user-attachments/assets/a008f8f1-79b5-437f-a90d-b8e35f77d409">
It should work if it's compatible. Basically most errors occur because of the API URL. Try using `https://api.deepseek.com/v1/` in the custom URL section.

**Reply by NielSync** [reply]
![Screenshot_20241103-132442.png](https://github.com/user-attachments/assets/eb9cbcf0-5898-4157-b86a-58c86a7c567b)

Used that before but errors.
Needs to be investigated further perhaps. Thanks!
